hw4/01-routing.js

```diff
@@ -31,18 +31,48 @@ let getRoutes = () => {
   return result;
 };
 
+
 app.get('/', (req, res) => {
   let routeResults = getRoutes();
 
   res.writeHead(200, { 'Content-Type': 'text/html' });
-  res.write(`<h1>Exercise 04</h1>`);
+  res.write('<h1>Exercise 04</h1>');
   res.write(`<ul> ${routeResults} </ul>`);
   res.end();
 });
 
-app.get('/welcome', (req, res) => {});
 
-// Add your code here
+app.get('/welcome', (req, res) => {
+  res.status(200).send('<h1>Welcome</h1>');
+});
+
+
+app.get('/redirect', (req, res) => {
+  res.redirect('/redirected');
+});
+
+
+app.get('/redirected', (req, res) => {
+  res.send('<h1>Redirected</h1>');
+})
+
+
+app.get('/cache', (req, res) => {
+  res.set('Cache-control', 'max-age=86400');
```

Comment:
it should have 'Content-Type': 'text/plain'

Reply:
Thank you!